CWE-079/examples/ReflectedXssGood.ts

```
@@ -0,0 +1,17 @@
+//{fact rule=cross-site-scripting@v1.0 defects=0}
+
+var escape = require('escape-html');
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)